DNN Platform/Library/Common/Utilities/DataCache.cs

```diff
@@ -211,6 +211,7 @@ public class DataCache
         public const string UserProfileCacheKey = "UserProfile|{0}|{1}";
         public const int UserProfileCacheTimeOut = UserCacheTimeOut;
 
+        public const string UserNotificationsConversationCountCacheKey = "UserNitifConversationCount|{0}|{1}";
```

Comment:
this cache seems useless with 2 points:
1. it will cause user won't receive notification even refresh page in 30 seconds;
2. if user stay on page, the ajax request every 30 seconds, that means cache will always expired in each request.